web/src/components/Button.tsx

```diff
@@ -5,6 +5,7 @@ import { makeStyles } from '@material-ui/core/styles'
 
 interface CustomProps {
   loading?: boolean
+  noSpinner?: boolean
```

Comment:
Maybe rename to hasSpinner? That way you don't have to have weird things like:`!noSpinner`?

Reply:
Done